jupyter-extension/jupyterlab_pachyderm/tests/test_integrations.py

```diff
@@ -424,6 +424,7 @@ def _update_metadata(notebook: Path, project_name: str, repo_name: str, pipeline
     config.pipeline = dict(name=pipeline_name, project=dict(name=project_name))
     # sub in repo_name
     config.input_spec = f"pfs:\n  repo: {repo_name}\n  glob: \"/*\""
+    config.resource_spec = "" # this is currently not being tested so it is set to the empty string
```

Comment:
If we're not testing now, let's create a ticket to add tests later?

Reply:
Done! https://pachyderm.atlassian.net/browse/INT-1057